content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -328,11 +318,9 @@ using default constraints for `PodTopologySpread`.
 
 #### Internal default constraints
 
-{{< feature-state for_k8s_version="v1.20" state="beta" >}}
+{{< feature-state for_k8s_version="v1.24" state="stable" >}}
 
-With the `DefaultPodTopologySpread` feature gate, enabled by default, the
-legacy `SelectorSpread` plugin is disabled.
-kube-scheduler uses the following default topology constraints for the
+Kube-scheduler uses the following default topology constraints for the
 `PodTopologySpread` plugin configuration:
```

Comment:
```suggestion
If you don't configure any cluster-level default constraints for pod topology spreading,
then then kube-scheduler acts as you specified the the following default topology constraints:
```

Reply:
`as if you`? I'm not a native speaker, so feel free to disregard if I made a mistake :)

Reply:
“as if you” is correct; sorry!

Reply:
thanks. updated.